src/app/(post-login)/submission/program/[shortName]/registration/components/FilePreview.tsx

```diff
@@ -77,7 +80,13 @@ const FilePreview = ({
     <FileTable
       records={records}
       stats={stats}
-      submissionInfo={submissionInfo}
+      submissionInfo={
```

Comment:
gql generated typings introduces a bunch of "Maybe" types. I haven't found a good way around this apart from type assertions